patches/tModLoader/Terraria/ID/ItemID.TML.cs

```diff
@@ -74,5 +74,20 @@ partial class Sets
 			{ Diamond, (3, 7) },
 			{ Amber, (3, 7) }
 		};
+
+		/// <summary>
+		/// Dictionary for defining what ores can spawn as bonus drop inside slime body. All items in this dictionary are equally likely to roll, and will drop with a stack size between minStack and maxStack (exclusive).
```

Comment:
This isn't exclusive, I don't think

Reply:
Yes, I know. So since 1 is added, a drop of 13 is possible, so the upper bound is inclusive, right? The wiki also says that they drop 3 to 13. not 3 to 12